articles/setting-up-cicd-for-python-packages-using-github-actions/index.md

```diff
@@ -0,0 +1,279 @@
+# Setting up CI/CD for Python Packages using GitHub Actions
+
+## Outline
+
+- Prerequisites
+- What is CI/CD
+- Building a Python Package
+- Authenticating GitHub with Test PyPI
+- Packaging & Deploying with GitHub Actions
+- Testing Python Package Locally
+- Conclusion
```

Comment:
```suggestion
- [Prerequisites](#prerequisites)
- [What is CI/CD](#what-is-cicd)
- [Building a Python Package](#building-a-python-package)
- [Authenticating GitHub with Test PyPI](#authenticating-github-with-test-pypi)
- [Packaging & Deploying with GitHub Actions](#packaging-deploying-with-github-actions)
- [Testing Python Package Locally](#testing-python-package-locally)
- [Conclusion](#conclusion)
```
It's a good practice to add redirecting links. It enables the reader to navigate through the page easily.